### DISCUSSION TITLE
Using Uno with firebase

### DISCUSSION THREAD

**Discussion by Marc-Antoine-Soucy** [unoplatform/uno#16959] [Q&A]
I would like to know what is the recommended way/ how I could add firebase to my uno application. For android, we can simply use the xamarin supported one, but for IOS, [the repo](https://github.com/xamarin/GoogleApisForiOSComponents/issues) is deprecated and has been archived by microsoft.

We started using [this repo](https://github.com/AdamEssenmacher/GoogleApisForiOSComponents), and it works well, sometimes, but some of the nugets are [not well tested](https://github.com/AdamEssenmacher/GoogleApisForiOSComponents/issues/4#issuecomment-2133940176) Since it is a repo made and supported by one person. Is there any good alternative?

**Comment by agneszitte** [comment]
fyi @jeromelaban we have these related documentation issues for a while now https://github.com/unoplatform/uno/issues/4124 and https://github.com/unoplatform/uno/issues/3661
